### PR TITLE
adding controler layer

### PR DESCRIPTION
**Pull request by umeshams**
please check

### CODE REVIEW

File: src/main/java/com/example/gittest/controler/UserControler.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class UserControler {
+
+	@GetMapping("/get")
+	public String getMesg() {
+		return "hi";
+	}
```
Comment (umeshams28) [inline src/main/java/com/example/gittest/controler/UserControler.java:12]: add onther method

**Comment by umeshams28** [comment]
and add product module

**Comment by umeshams** [comment]
hi i added method

**Comment by umeshams28** [comment]
ya do marge

**Pull request merged**